### ISSUE TITLE
No sound from speakers and headphones with pop-os-linux while old patch worked

### ISSUE THREAD

**Issue by daevilzorg** [codepayne/pop-os-linux#9]
On my Matebook 14 the "old" version of your patch is running without major problems. I even applied your patch to newer kernel versions and it still does it's job ;-) I too really appreciate your efforts! 

My system:
System Information
        Manufacturer: HUAWEI
        Product Name: KLVL-WXXW
        Version: M1010
        Serial Number: PRJBB22224800650
        UUID: 20220226-a497-b115-3069-a497b115306a
        Wake-up Type: Power Switch
        SKU Number: C100
        Family: MateBook 14

With pop-os-linux kernel the hardware obviously gets recognized by the new driver but no sound is coming out of neither my speakers nor the headphones. No matter how high I turn the controls...
Anyone else facing this problem? Am I missing something??

**Comment by codepayne** [comment]
Do you still have the problem?

**Comment by daevilzorg** [comment]
Yes I do. Well actually I am using the "old" patch for daily work. I am running a rolling release (Manjaro Linux) so every now and then I apply your patch to an newer 6.1.x kernel to keep up with the release kernel version.
I have not given it another try with pop-os-kernel yet...

**Comment by codepayne** [comment]
If you want you can attach a screenshot of alsamixer settings. Or you can wait for kernel 6.7 that should have the sound driver officially inside.
